model.go

```diff
@@ -26,11 +39,146 @@ func LoadModelFile(path string) (*Model, error) {
 	return LoadModel(fp)
 }
 
-func LoadModel(r io.Reader) (*Model, error) {
+func LoadModel(r io.Reader) (Model, error) {
```

Comment:
__[gometalinter]__
* [golint] exported function LoadModel should have comment or be unexported